### DISCUSSION TITLE
Default block per core field type

### DISCUSSION THREAD

**Discussion by ssyberg** [payloadcms/payload#384] [Feature Requests & Ideas]
Perhaps there's already a solve for this, but would be great to include a ready to go block for each core field type such that it would be trivial to add a `richText` or `text` field to `blocks`

**Comment by jmikrut** [comment]
Hey @ssyberg — this is a great idea! Within our own Payload blocks we've build just about any block config you can think of. They could definitely be of relevance to be shared publicly.

But for many developers, block configs are typically designed specifically to suit the project design—i.e. in our own projects, our blocks are frequently never as simple as just a `richText` or `text` field. Instead, they might have other options, like `backgroundColor`, `columns`, or similar. 

For example, you could take a look at our walkthrough called [Building a Professionally Designed Website with NextJS and TypeScript](https://payloadcms.com/blog/building-professionally-designed-site-nextjs-typescript-episode-1).  The resulting project is [open source](https://github.com/payloadcms/custom-website-series) and shows some of the block configs in use to produce that design and specifically the [`Content`](https://github.com/payloadcms/custom-website-series/blob/master/blocks/Content/index.ts) block may be of relevance.

Last note - on our [Roadmap](https://payloadcms.com/roadmap), we have an item called [Improved Blocks UI](https://github.com/payloadcms/payload/discussions/288), which will give developers a way to completely overhaul the entirety of the blocks UI—at which point, we will _**absolutely**_ be compiling a list of pre-designed blocks for easy reuse by developers. That is going to be awesome.

Keep an eye out and let me know if this makes any sense!
